x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_utility_bar/index.tsx

```diff
@@ -260,3 +222,63 @@ export const AlertsUtilityBar = React.memo(
     prevProps.showBuildingBlockAlerts === nextProps.showBuildingBlockAlerts &&
     prevProps.showOnlyThreatIndicatorAlerts === nextProps.showOnlyThreatIndicatorAlerts
 );
+
+export const AditionalFiltersAction = ({
```

Comment:
👌🏾 Thanks for making this it's own component. I would just say it might be useful in the future to separate each of these filters into their own files, and then export each of them from the index so this file doesn't creep up in size